packages/serverpod/lib/src/database/value_encoder.dart

```diff
@@ -20,6 +20,22 @@ class ValueEncoder extends PostgresTextEncoder {
           escapeStrings: escapeStrings);
     } else if (input is UuidValue) {
       return "'${input.uuid}'";
+    } else if (input is Enum) {
```

Comment:
Chore: Add an e2e test for both enum types.

This is a tricky one, but we should test our golden path for serialization of both enum types to the client to ensure that everything is working as expected.